packages/html-program-viewer/package.json

```diff
@@ -23,6 +23,7 @@
       "default": "./dist/emitter/index.js"
     },
     "./react": {
+      "development": "./src/react/index.ts",
```

Comment:
Those allow vite to load the source of dependencies directly so you don't have to build and watch many packages